### ISSUE TITLE
not detecting other environments 

### ISSUE THREAD

**Issue by erwin16** [ahmadnassri/autoenv#1]
I have set up, .env , .env.test, .env.development files

In my index.js, I also set
```
require('autoenv');
console.log('Detected process.env,: ', process.env);
```

but when I start my tests with 👍 

`"test:v1": "cross-env NODE_ENV=test ./node_modules/.bin/mocha --ui bdd --reporter spec --colors --compilers js:babel-core/register server/v1/tests --recursive",`

then the console log displays the default .env, not the .env.test variables

```
Detected process.env,:  { npm_package_engines_npm: '>=2.15.11',
....
NODE_ENV: 'test',
...
  PORT: '4040',
  MONGO_USER: 'superAdmin',
  MONGO_PWD: 'superAdmin123',
  MONGO_HOST: 'localhost',
  MONGO_DB: 'mydb-api-development',
  MONGO_PORT: '7272' }

```
II should get the user, pwd, db for the test environment, not the development...
what could be wrong ?

**Comment by ahmadnassri** [comment]
it looks like `mocha` cli is not passing through the env values to tests

**Comment by ahmadnassri** [comment]
fixed in `v1.0.1`
